src/input/ButtonConfigWidget.cc

```diff
@@ -61,6 +62,12 @@ void ButtonConfigWidget::init() {
 			initComboBox(box,*ent);
 		}
 	}
+
+	WheelIgnorer *wheelIgnorer = new WheelIgnorer();
+	QList<QComboBox *> widgets = this->findChildren<QComboBox *>();
```

Comment:
This is another place where `auto` is nice: `auto widgets = this->findChildren<QComboBox *>();`

Reply:
using auto also allowed me to name the variable "comboBoxes" instead of widgets, without repeating the same word three times. I start to see the benefits of auto.

Reply:
Another side effect of this is that you don't even need to store the widgets as a temporary variable, you could iterate directly; `for (auto comboBox : this->findChildren<QComboBox *>())  {..`